sv_pipeline/exome/load_data.py

```diff
@@ -89,12 +105,19 @@ def _get_seqr_sample_id(raw_sample_id, sample_type='WES'):
     },
     GENES_COL: {
         'field_name': GENES_FIELD,
-        'format': lambda genes: set() if genes == 'None' else {gene.split('.')[0] for gene in genes.split(',')},
+        'format': _parse_genes,
     },
+    PREV_IDENTICAL_COL: {'field_name': 'prev_call', 'format': _parse_prev_call},
```

Comment:
As suggested in the other PR, how about using `prev_identical` rather than `prev_call`?

Reply:
no I think its clearer to state that this is previously called, because that what it means. Identical is ambiguous outside of the file - identical to what? and I want to use short field names here which makes `prev_identical_call` and the like too long to use